openwisp_controller/config/static/config/js/widget.js

```diff
@@ -153,6 +153,29 @@
         return error;
     };
 
+    var validateMaxLength = function(){
+        var config_variables = {};
+        if (window.getContext()){
+            config_variables = JSON.parse(window.getContext().value);
+        }
+        var config_variables_keys = Object.keys(config_variables);
+        var all_fields = $(".jsoneditor .vTextField");
+        all_fields.map((index, field)=>{
+            if ($(field).attr('maxLength')){
+                var maxLength = $(field).attr('maxLength');
+                $(field).on('input', () => {
+                    $(config_variables_keys).map((i,key)=>{
+                        if (key.startsWith($(field).val()) || $(field).val().startsWith('{')){
```

Comment:
I do not understand this. Does this take into account also when the variable is in the middle of the content?
Why not use something like `$(field).val().indexOf('{{') > -1`?

Reply:
I thought curly braces will always be in starting :sweat_smile: . I will fix this.